bindings/netframework/ElectionGuard.NetFramework/ElectionGuardInterop/Group.h

```diff
@@ -0,0 +1,106 @@
+#pragma once
+
+#include "constants.h"
+#include "group.hpp"
+#include "ManagedInstance.h"
+using namespace System;
+using namespace System::Runtime::InteropServices;
+
+namespace ElectionGuardInterop
+{
+  public
+    ref class ElementModP : ManagedInstance<electionguard::ElementModP>
+    {
+      public:
+        ElementModP(array<uint64_t> ^ elem)
```

Comment:
any external calls are expected to pass in a managed object, which we then copy into a native object and pass to the library (which then makes ANOTHER copy internally).  This is a completely superfluous copy operation, but it is unavoidable without polluting the core c++ native library with platform-specific code just to support this use case.  If performance becomes an issue, we can revisit.